src/components/Modal/CalendarSave/CalendarCaptureCard.tsx

```
@@ -79,7 +82,19 @@ export function CalendarCaptureCard({ year, month, selectedBg, ootdListData }: C
                   </Text>
                 </View>
                 <View style={styles.imageWrapper}>
-                  {ootdData?.image ? (
+                  {ootdData?.items != null &&
+                  ootdData?.canvasSize != null &&
+                  ootdData.items.length > 0 ? (
+                    <View style={[styles.ootdImage, { width: CAPTURE_PREVIEW_SIZE, height: CAPTURE_PREVIEW_SIZE }]}>
```

ESLint

[error] Replace `·style={[styles.ootdImage,·{·width:·CAPTURE_PREVIEW_SIZE,·height:·CAPTURE_PREVIEW_SIZE·}]}` with `⏎······················style={[⏎························styles.ootdImage,⏎························{·width:·CAPTURE_PREVIEW_SIZE,·height:·CAPTURE_PREVIEW_SIZE·},⏎······················]}⏎····················`

(prettier/prettier)